src/builtin.jq

```diff
@@ -249,17 +249,18 @@ def bsearch($target):
       else .[2]
       end
   end;
-
+#
```

Comment:
I prefer this line to be an empty line rather than an empty comment.

Reply:
OK, but someone should probably run: `s/^# *$//`